ansible/mnesia_snapshot_restore.yml

```diff
@@ -36,9 +36,9 @@
       tags: [health-check]
 
     - name: Restore Mnesia database
-      command: "{{ aeternity_bin }} eval '{atomic, _} = mnesia:restore(\"/tmp/backup\", [{default_op, recreate_tables}]).'"
```

Comment:
Also I think we also need to somehow pass `recreate_tables`, otherwise the backup's schema is not considered during the restore.
While this should be fine on restoring on new instances, it's a problem when the database is already there and have non-compatible schema.

As we use this only on new nodes, maybe track this for further improvement if not's trivial to pass the option in the PR.